torch/csrc/jit/codegen/cuda/iter_visitor.cpp

```diff
@@ -479,7 +479,7 @@ struct FindOutputs : public IterVisitor {
 
   FindOutputs(const std::unordered_set<Val*>& _of) : of_(_of) {
     auto fusion = (*of_.begin())->fusion();
-    traverse(fusion);
+    traverseFrom(fusion, fusion->outputs(), true);
```

Comment:
Needed to make sure all outputs of a val are grabbed. This could be done much more efficiently with a simpler pass than iter visitor, just traversing through uses.

Reply:
Makes sense.